### ISSUE TITLE
dragImage position error

### ISSUE THREAD

**Issue by pedoc** [chakming/HTML5-Chinese-Calligraphy#2]
hi,it's amazing effect,very cool!
in your example,it's work fine
#canvas {
left:0;
top:0;
}
but,when i set canvas element style as follow，will not work properly.
#canvas {
left:100px;
top:100px;
}
Is there any solution? thanks

**Comment by pedoc** [comment]
i fixed self
```
  drawStroke: function (x1, y1, x2, y2, brushEnded) {
//adjust canvas offset here
        x1-=this.config.offset.x;
        y1-=this.config.offset.y;
        x2-=this.config.offset.x;
        y2 -= this.config.offset.y;

        //console.log('draw offset:', this.config.offset);
        /// <summary>Draw stroke line (Spline).</summary>
        if (x2 == 0 && y2 == 0) // tcuch screen hard code hardcode prevention debug
            return;
...
```

**Comment by chakming** [comment]
welcome PR :)

**Comment by pedoc** [comment]
@chakming thanks fot lot,

**Comment by chakming** [comment]
merged #4
